README.md

```diff
@@ -634,6 +634,13 @@ Or via [Chocolatey](https://chocolatey.org):
 choco install fd
 ```
 
+### On GuixOS
+
+You can install [the fd package](https://guix.gnu.org/en/packages/fd-8.1.1/) from the official repo:
```

Comment:
Is it possible to link to something that doesn't include a particular version number?

Reply:
Unfortunately, I don't see any way how to do it.